packages/core/lib/command-utils.js

```diff
@@ -206,9 +212,85 @@ const displayGeneralHelp = () => {
     .showHelp();
 };
 
+const getConfiguredNetworkUrl = function (customConfig, isDashboardNetwork) {
```

Comment:
Would it be slightly more flexible if the second argument were more general, like just `urlSuffix`? It probably does't matter too much in this case though.